server/sonar-server/src/test/java/org/sonar/server/computation/queue/CeQueueCleanerTest.java

```diff
@@ -49,10 +45,9 @@
   @Rule
   public TemporaryFolder tempFolder = new TemporaryFolder();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "TemporaryFolder". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2924)